src/routes/api/user.router.js

```diff
@@ -3,12 +3,14 @@ import UserController from '../../controllers/user.controller';
 import verifyToken from '../../middlewares/jwtAuth';
 import UserMiddleware from '../../middlewares/user.middleware';
 import roleValidation from '../../validation/user/role.validation';
+import Validator from  '../../validation/index';
 
 const { isSuperAdmin } = UserMiddleware;
-const { setUserRole, getManagers } = UserController;
+const { setUserRole, getManagers, retrieveProfile } = UserController;
 const router = express.Router();
 
 router.patch('/role', verifyToken, isSuperAdmin, roleValidation, setUserRole);
 router.get('/managers', verifyToken, getManagers);
+router.get('/user/profile', verifyToken, Validator.rememberMe, retrieveProfile);
```

Comment:
`Validator.rememberMe` seems will throw a `400` error if the user chose to not be remembered. If yes, should this endpoint behave in this way?

![Screenshot (92)](https://user-images.githubusercontent.com/36412651/64498494-a7076b80-d2ac-11e9-8510-fd71ef91f4af.png)


Also, I think there is need to seek clarity on when the user makes the `rememberMe` choice. Is it when signing up, signing in, or when making a request?

Reply:
Read the PT story. Thanks

Reply:
Ok. Thanks